### ISSUE TITLE
CI can no longer invoke h2spec

### ISSUE THREAD

**Issue by hawkw** [carllerche/h2#158]
Because today _had_ to be the day of everything on CI breaking :/

It looks like invoking the `h2spec` executable now [fails on Travis](https://travis-ci.org/carllerche/h2/jobs/289728104#L1168) with the message "cannot execute binary file: Exec format error". I suspect this is probably because the `h2spec` tarball we're fetching is no longer valid?

**Comment by hawkw** [comment]
Okay, this is wacky: the error message ` Exec format error` implies that the binary is formatted for the wrong system architecture, but we're downloading `h2spec_linux_amd64.tar.gz`, and the CI box is [still reporting that it's `linux/amd64`](https://travis-ci.org/carllerche/h2/jobs/289728104#L40) as expected. 

I also ran `file h2spec` on the downloaded executable on my machine, just to verify that it's correct:
```
$ file h2spec 
h2spec: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, not stripped
```

If someone with a linux box wants to manually verify that the h2spec v2.1.0 linux/amd64 executable can still be invoked, that would be much appreciated!

**Comment by hawkw** [comment]
Huh, using an earlier h2spec version [does not fix this](https://travis-ci.org/carllerche/h2/jobs/289744686). Unsure why it only seems to be happening on my branch?

**Comment by hawkw** [comment]
Tested `wget`ing the same file manually on a linux box and the h2spec executable seems to work fine...

**Comment by hawkw** [comment]
Never mind, this is entirely my fault. Seems I accidentally committed a h2spec Mac executable to that branch. Fixed in 2d53cf0112b8a68553d158a7648b3c7095b44518.
